jupyter_server_proxy/handlers.py

```diff
@@ -330,7 +334,11 @@ def proxy_request_headers(self):
     def proxy_request_options(self):
         '''A dictionary of options to be used when constructing
         a tornado.httpclient.HTTPRequest instance for the proxy request.'''
+<<<<<<< HEAD
```

Comment:
@rcthomas this might be your culprit, pesky git 😄

Reply:
Oh bother, thought I got them all...